src/hound/cmds/houndd/main.go

```diff
@@ -190,14 +191,13 @@ func makeSearchers(
 			s, err = searcher.New(path, repo)
 		}
 
-		if err != nil {
-			return nil, err
+		if err == nil {
+			m[strings.ToLower(name)] = s
 		}
-		m[strings.ToLower(name)] = s
 
 	}
 
-	return m, nil
+	return m, err
```

Comment:
Yeah, although currently we don't actually display this error anywhere. Previously it was just "exit status 1" anyway.